src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue3139.cs

```diff
@@ -13,15 +12,23 @@ public Issue3139(TestDevice testDevice) : base(testDevice)
 
 	public override string Issue => "DisplayActionSheet is hiding behind Dialogs";
 
-	// [Test]
-	// [Category(UITestCategories.ActionSheet)]
-	// public void Issue3139Test ()
-	// {
-	// 	App.WaitForElement (q => q.Marked ("Click Yes"));
-	// 	App.Tap (c => c.Marked ("Yes"));
-	// 	App.WaitForElement (q => q.Marked ("Again Yes"));
-	// 	App.Tap (c => c.Marked ("Yes"));
-	// 	App.WaitForElement(q => q.Marked("Test passed"));
-	// }
-}
-#endif
\ No newline at end of file
+	 [Test]
+	[Category(UITestCategories.ActionSheet)]
+	public void Issue3139Test()
```

Comment:
This test is failing on iOS:
<img width="1139" alt="image" src="https://github.com/user-attachments/assets/850c4955-2bd5-4469-90e4-6bb838e00535">

Reply:
Resolved and committed changes, kindly review and share your concerns.